subcommands/docker/cmd.go

```diff
@@ -0,0 +1,117 @@
+package docker
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+
+	"github.com/foundriesio/fioctl/subcommands"
+	"github.com/mitchellh/go-homedir"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const DOCKER_CREDS_HELPER = "docker-credential-fio"
+
+var (
+	dockerConfigFile string
+	helperPath       string
+)
+
+func NewCommand() *cobra.Command {
+	path, err := exec.LookPath("docker")
+	subcommands.DieNotNil(err, "Docker not found on system")
+	helperPath = filepath.Dir(path)
+
+	dockerConfigFile = dockerConfigPath()
+
+	cmd := &cobra.Command{
+		Use:   "configure-docker",
+		Short: "Configure a hub.foundries.io Docker credential helper",
```

Comment:
Some bigger help message about how to use the `creds-path` or run this in sudo mode would be helpful.

Reply:
added more in the force-push - see what you think